.github/workflows/bump-version.yml

```
@@ -0,0 +1,33 @@
+name: Versioning
+
+on:
+  pull_request:
+
+jobs:
+  bump-version:
+    name: Check version has been bumped
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout to BASE
+        uses: actions/checkout@v4
+        with:
+          ref: dev
+
+      - name: Get BASE_VERSION
+        run: |
+          echo "BASE_VERSION=$(grep '"version"' apps/web/package.json | sed -E 's/.*"version": *"([^"]+)".*/\1/')" >> $GITHUB_ENV
```

We can assume that there will always be a single space

```suggestion
          echo "BASE_VERSION=$(grep '"version"' apps/web/package.json | sed -E 's/.*"version": "([^"]+)".*/\1/')" >> $GITHUB_ENV
```